pynn_spinnaker/simulator.py

```diff
@@ -340,6 +351,9 @@ def _load_current_input_verts(self, placements, allocations,
                 # Select placed chip
                 with self.machine_controller(x=vertex_placement[0],
                                              y=vertex_placement[1]):
+                    # Configure watchdog for this vertex
+                    self._configure_software_watchdog()
```

Comment:
This only needs doing once per chip rather than once per core. Maybe just a loop over all chips setting this would be easier?